frontend/src/helpers/chemical-formatters.js

```diff
@@ -17,4 +17,15 @@ export default function chemicalFormula(formula, charge) {
     form = form.replace(/([-+])$/, '<sup>$1</sup>');
   }
   return form;
-}
+};
+
+const chemicalEquation = formula => {
+  let equation = formula.replace('<=>', '⇔');
+  equation = equation.replace('=>', '⇒');
+  equation = equation.replaceAll(/(\S)\+/g, '$1<sup>+</sup>');
+  equation = equation.replaceAll(/(\S)-\s/g, '$1<sup>-</sup> ');
+  equation = equation.replaceAll(/([A-Z])(\d+)/g, '$1<sub>$2</sub>');
```

Comment:
This may bring in some unwanted formatting, e.g. for the reaction `R00073`, the number 2 for the compound `Prostaglandin G2` should not be `sub`. Basically, only a few chemicals such as H2, O2 and H2O should be formatted but it is hard to include everything wanted while excluded unwanted. Maybe we can just ignore this line of formatting.

Reply:
It seems that we may have the same problem in the GEM Browser, for example: `/explore/Human-GEM/gem-browser/reaction/MAR04223`. If this is the case, I'm ok with it being wrong in the same way in the GotEnzymes page for now, then we could create a new bug issue to track this problem for both pages.

<img width="174" alt="image" src="https://user-images.githubusercontent.com/423498/191242147-b0aaa200-6c52-46ad-9b62-32e38823d299.png">

Reply:
That's tricky! And a nice observation that we have the same problem in the GEM Browser. To me, keeping it and creating a new issue would make sense then, if @nanjiangshu does not object.

Reply:
I'm OK with it. Let's keep it as is and create a new issue to solve it for both here and the GEM browser.

Reply:
I created #1140 for this. Please fill in more examples if you know of any.